### PR TITLE
doc: documenting the feature that scrub clear the entries from damage…

### PR DESCRIPTION
**Pull request by neesingh-rh**
… table on repair

Fixes: https://tracker.ceph.com/issues/67337
Signed-off-by: Neeraj Pratap Singh <neesingh@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/cephfs/scrub.rst [hunk 1]
```diff
@@ -143,6 +143,14 @@ The types of damage that can be reported and repaired by File System Scrub are:
 
 * BACKTRACE : Inode's backtrace in the data pool is corrupted.
 
+The above named MDS damages can be repaired by using the following command::
```
Comment (joscollin) [inline doc/cephfs/scrub.rst:146]: s/The above named/These

**Review by zdover23 — APPROVED** [review]
LGTM

File: doc/cephfs/scrub.rst [hunk 2]
```diff
@@ -143,6 +143,14 @@ The types of damage that can be reported and repaired by File System Scrub are:
 
 * BACKTRACE : Inode's backtrace in the data pool is corrupted.
 
+These above named MDS damages can be repaired by using the following command::
+
+    ceph tell mds.<fsname>:0 scrub start /path recursive, repair, force
+
+If the damage is repaired using the above scrub command, there is no need to manually
+delete the damage entry.
```
Comment (vshankar) [inline doc/cephfs/scrub.rst:151]: Reword as: If scrub is able to repair the damage, the corresponding entry is automatically removed from the damage table.

Comment (anthonyeleven) [reply doc/cephfs/scrub.rst:151]: Agree

Comment (neesingh-rh) [reply doc/cephfs/scrub.rst]: Addressed

**Review by vshankar — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/59078 - Squid backport
https://github.com/ceph/ceph/pull/59079 - Reef backport
https://github.com/ceph/ceph/pull/59080 - Quincy backport

**Comment by neesingh-rh** [comment]
> #59078 - Squid backport #59079 - Reef backport #59080 - Quincy backport

Thanks!! @zdover23 Can pls update the respective existing backport trackers with PR ids and update the status of tracker.

**Pull request merged**